README.md

```diff
@@ -42,7 +45,7 @@ Refer to [Github documentation](https://help.github.com/articles/cloning-a-repos
  4. You could generate expected result files using Postgres or any such database.
  5. In the `framework/resources/Datasources` directory, create corresponding datasource directories and copy over any required scripts and datasets required by the tests.
 
-#### Structure of test definiton files
+#### Structure of test definiton files. A test definition file defines one or a set of tests in json format. The file name must ends with .json. Framework looks for files with .json extension in specified location(s) and pick up all test(s) defined by them. 
```

Comment:
Can you please separate this into two lines. 

#### Structure of test definition files. 
A test definition file is a JSON file that defines a test suite, which is a set of tests within a directory. The framework scans for files with .json extension in the specified location(s) and executes all test(s) defined by it.